.github/workflows/self-service-labeler.yml

```diff
@@ -0,0 +1,40 @@
+name: Issue Opened Self-Service Labeler
+
+on:
```

Comment:
We're you able to do any testing with this? It might be helpful for testing/one-offs to add a manual option where the user provides an issue id (https://docs.github.com/en/actions/using-workflows/events-that-trigger-workflows#providing-inputs). No worries if that would make more sense in a separate PR, I just brought it up here for the testing benefit.

Reply:
I tested in my fork here: https://github.com/trodge/terraform-provider-google/issues

Reply:
👍 awesome thanks!